pkgs/by-name/ci/cinzel/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenvNoCC, fetchFromGitHub, rename }:
+
+stdenvNoCC.mkDerivation {
+  pname = "cinzel";
+  version = "dd59849";
```

Comment:
```suggestion
  version = "0-unstable-2020-07-22";
```

Reply:
Thank you, updated it.

Reply:
Also should update the PR title :p

Reply:
Where did you get this version string from?

Reply:
I cannot find this proposed version string anywhere